### DISCUSSION TITLE
What component to use for pagination (Prev vs. Next)

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#938] [Q&A] [answered]
Hello,

What component paste (anchor vs. link button) may I use for pagination for Prev / Next?
If anchor, then how do/can I show prev and next anchor as disabled for the first and last page?

![pagination](https://user-images.githubusercontent.com/69016180/99127926-58dfad80-25be-11eb-9a8f-895596829ba4.png)

If we use the link button 

<img width="290" alt="pagination-link-button" src="https://user-images.githubusercontent.com/69016180/99133040-3acd7980-25cd-11eb-8a4e-eeba0c77c6bd.png">

**Comment by kate-porter** [comment, marked as answer]
Hi @ishah-twilio -

Does each page in the table have its own URL? If so, then you should use Anchor for "Next" and "Previous". If each page does NOT have its own URL, then you should use Button.

If you end up using Button, then you can just use a disabled button when the user is on the first or last page.

If you end up using Anchor, I would suggest just hiding the "Next" or "Previous" link when the user is on the first or last page. Links / Anchors cannot have a `disabled` state, so that's probably the easiest approach.

**Comment by ishah-twilio** [comment]
@katieporter

Pagination will have its own url. So Next and Previous will be anchor component and will hide the link on first and last page.

Thank you.

**Reply by kate-porter** [reply]
Sounds good!

Also, I think the timeline you are working on it too tight to wait for this, but we will have a pagination component in our next release on December 15. If you are able to wait for that release, that's great, but otherwise, doing pagination as we've discussed is fine.

**Reply by ishah-twilio** [reply]
Thank you @katieporter 

We don't start development on this feature till next year. So we will use the DSys component that will be released on December 15.
